README.md

```
@@ -2,6 +2,28 @@
 # Chords-LSL-Connector
 Rust based LSL connector for device running Chords Firmware.
 
+# Installation Instructions
+### macOS  
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)